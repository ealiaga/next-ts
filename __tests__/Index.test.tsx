import * as React from 'react';
import * as TestRenderer from 'react-test-renderer';
import Index from '../pages/Index';

describe('Index', () => {
  it('should create instance', () => {
    const component = TestRenderer.create(<Index />); // React.createElement('Index', null, null)
    expect(component.toJSON()).toMatchSnapshot();
  });
});
