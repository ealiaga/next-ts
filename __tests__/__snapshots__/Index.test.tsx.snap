// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index should create instance 1`] = `
<div>
  Hola mundo
</div>
`;
